src/index.js

```diff
@@ -326,6 +327,67 @@ function crossEnv(args) {
   return `${getBin('cross-env')} ${args}`
 }
 
+/**
+ * The options to pass to includePackage
+ * @typedef {Object|string} IncludePackageOptions
+ * @property {string} path - the path to the package scripts
+ */
+
+/**
+ * Includes the scripts from a sub-package in your repo (for
+ * yarn workspaces or lerna style projects).
+ * @param {IncludePackageOptions} packageNameOrOptions - either a
+ * simple name for the sub-package or the relative path to it.
+ *  If you just provide the name and not the path, then the path
+ *  defaults to: /packages/{package}/package-scripts.js
+ * @return {any} will return an object of scripts loaded from that package
+ */
+function includePackage(packageNameOrOptions) {
+  const packageScriptsPath = typeof packageNameOrOptions === 'string' ?
+    `./packages/${packageNameOrOptions}/package-scripts.js` :
+    packageNameOrOptions.path
+  
+  const startingDir = process.cwd()
+  const relativeDir = path.relative(startingDir,
+    path.dirname(packageScriptsPath)).split('\\').join('/')
+  const relativeReturn = path.relative(relativeDir, startingDir)
+
+  let scripts = null
+  try {
+    scripts = require(packageScriptsPath)
+  } catch (e) {
+    throw Error(commonTags.oneLine`Couldnt include the package, 
```

Comment:
This is helpful to have a good error message, but let's only throw it if we're sure the reason the original error was thrown is because the file does not exist (maybe do a require.resolve first or check the type of error). I've been bitten by this in the past where the error thrown was actually a syntax error in the file and I spent a lot of time trying to figure out why it couldn't find my file when it turned out that it could and the error was a red herring.

Reply:
Oh thats a very good point. Ill write some tests for syntax errors too.

Reply:
Okay unfortunately require.resolve doesnt work with jest mocks, so decided that the extra complication probably isnt worth it and the native errors are good enough.